config/routes.rb

```diff
@@ -0,0 +1,14 @@
+Rails.application.routes.draw do
+  devise_for :users
+
+  unauthenticated :user do
+    root to: 'users#index'
+  end
+
+  authenticated :user do
+    root 'groups#index', as: :authenticated_root
+  end
+
+  resources :groups
+  resources :expenses
+end
```

Comment:
- [ ] Please implement a Splash screen and set it as a root route that has two buttons (login and sign up), as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=Splash%20screen,log%20in%20pages.) requirement.